src/android/CameraLauncher.java

```diff
@@ -1291,11 +1291,12 @@ public void processPicture(Bitmap bitmap, int encodingType) {
 
         try {
             if (bitmap.compress(compressFormat, mQuality, jpeg_data)) {
+                StringBuilder sb = new StringBuilder();
+                sb.append("data:image/jpeg;base64,");
```

Comment:
I believe the `encodingType` that is passed into `processPicture` could be either `JPG` or `PNG`.

Maybe use `getMimetypeForEncodingType` to get the correct mime type?  It should return `PNG_MIME_TYPE` (`image/png`) for `PNG` or `JPEG_MIME_TYPE` (`image/jpeg`) for `JPEG`.

Reply:
lol, i just assumed it was jpeg since the variable was called jpeg_data... but yah it does seem to potentially be png